pettingzoo/classic/chess/chess_utils.py

```diff
@@ -321,17 +324,18 @@ def get_observation(orig_board, player):
       }
     """
     # from 0-63
-    square = board.ep_square
+    # Adjust the row number for the white pawn to the 8th if the en passant flag is set, and vice versa for black pawns.
+    square = board.ep_square  # (int) where the en passant happened
```

Comment:
Minor but probably most official sounding to say `# square where en passant happened (int)`

Reply:
Sure, I update this comment.